functions/utils/getSetup.ts

```diff
@@ -0,0 +1,15 @@
+const watermark = 'https://app.uniswap.org/images/324x74_App_Watermark.png'
+const check = 'https://app.uniswap.org/images/54x54_Verified_Check.svg'
+
+export default async function getSetup() {
+  const font = fetch(
+    new Request('https://fonts.gstatic.com/s/inter/v12/UcCO3FwrK3iLTeHuS_fvQtMwCp50KnMw2boKoduKmMEVuI6fAZFhjQ.ttf')
```

Comment:
Is there a reason you can't use our own copy, or store our own ttf copy in the public folder?
https://github.com/Uniswap/interface/blob/main/public/fonts/Inter-roman.var.woff2

Reply:
oh shoot i didn't even know that existed LOL

Reply:
just tried... og only accepts `ttf` file types